packages/machine/src/protocol/install.ts

```diff
@@ -147,7 +147,7 @@ function constructInstallOp(
     freeBalance.hashOfLatestState,
     freeBalance.nonce,
     freeBalance.timeout,
-    freeBalance.appSeqNo,
+    app.appSeqNo,
```

Comment:
I think this was meant to be `freeBalance.appSeqNo`.

Reply:
Why?

Reply:
It goes into `conditionalTransactionInput`